data/styles/default/include/Basemap.mapcss

```diff
@@ -155,6 +155,13 @@ area|z16-[landuse=forest],
 area|z16-[leisure=garden],
 {fill-color: @forest;}
 
+area|z12-[leisure=garden][garden:type=residential],
+{fill-color: @green5; fill-opacity: 0.4;}
```

Comment:
Why not use the same color as regular `[leisure=garden]` but make it less opaque?
Or even use green1/2/3... colors instead of green3/4/5...?

Landuse areas are often overlayed on top of each other, so relying a lot on transparency might lead to unpredictable mixtures esp. if a semi-translucent landuse is mapped on top of a more dark color.

WDYT?